scripts/tasks/check.ts

```diff
@@ -4,8 +4,8 @@ import { maxConcurrentTasks } from '../utils/maxConcurrentTasks';
 
 const parallel = process.env.CI ? 8 : maxConcurrentTasks;
 
-const linkCommand = `nx run-many --target="check" --all --parallel=${parallel} --exclude=@storybook/addon-storyshots,@storybook/addon-storyshots-puppeteer,@storybook/vue,@storybook/svelte,@storybook/vue3,@storybook/angular`;
-const nolinkCommand = `nx run-many --target="check" --all --parallel=${parallel} --exclude=@storybook/addon-storyshots,@storybook/addon-storyshots-puppeteer`;
+const linkCommand = `nx run-many --target="check" --all --parallel=${parallel} --exclude=@storybook/addon-storyshots,@storybook/addon-storyshots-puppeteer,@storybook/vue,@storybook/svelte,@storybook/vue3,@storybook/angular,root && nx run root:check:scripts`;
+const nolinkCommand = `nx run-many --target="check" --all --parallel=${parallel} --exclude=@storybook/addon-storyshots,@storybook/addon-storyshots-puppeteer,root && nx run root:check:scripts`;
```

Comment:
Here we could avoid using two commands with:
```
'nx run-many --target="check","check:scripts" --all <...>'
```
But I think we'd need to rename the "check" script in `code/package.json` to make that work.